src/demos/calcite-flow.html

```diff
@@ -51,8 +51,14 @@ <h1>calcite-flow</h1>
             margin: 1rem 0.2rem;
             min-width: 5rem;
           }
+          .toggles {
+            margin: 1rem 0;
+          }
```

Comment:
should be able to move this to `src/demos/demos.css` to share across all demo files.

Reply:
I think we could but I also think this will get stripped out when we use the component @jcfranco is cooking up.